packages/@vue/cli-plugin-e2e-webdriverio/generator/template/wdio.local.conf.js

```diff
@@ -0,0 +1,21 @@
+const { config } = require('./wdio.shared.conf')
+
+exports.config = {
+  /**
+   * base config
+   */
+  ...config,
+  /**
+   * config for local testing
+   */
+  maxInstances: 1,
+  services: ['chromedriver'],
+  capabilities: [{
+    browserName: 'chrome',
+    'goog:chromeOptions': {
+      args: process.argv.includes('--headless')
+        ? ['--headless', '--disable-gpu']
+        : []
+    }
+  }]
+}
```

Comment:
Could you add a firefox example if we are going to allow choosing the browser on project creation?

Reply:
Added Firefox browser to the setup